homepage/homepage/content/docs/server-side/setup.mdx

```diff
@@ -117,3 +117,36 @@ Currently, the Jazz Loader is tested on the following edge environments:
 
 - Edge runtime environment that supports WebAssembly
 - `jazz-tools/load-edge-wasm` must be imported before any Jazz import
+
+
+## Node-API
+
+Jazz uses a WASM-based crypto implementation for maximum compatibility and performance across a wide variety of environments. 
+
+If you’re in full control of your runtime and know it will always run on Node.js with a supported hardware configuration, you can enable the faster native Node.js crypto implementation instead.
+
+You can use it as follows:
+
+<CodeGroup>
+```ts
+import { startWorker } from "jazz-tools/worker";
+import { NapiCrypto } from "jazz-tools/worker/napi";
```

Comment:
How about just `jazz-tools/napi-crypto`

Reply:
At this point, it's better to use `jazz-tools/napi`. In that way, we could export other types of NAPI-related features in the future.